lib/internal/bootstrap/node.js

```diff
@@ -377,14 +376,6 @@ function setupProcessObject() {
     configurable: false,
     value: 'process'
   });
-  // Make process globally available to users by putting it on the global proxy
-  ObjectDefineProperty(globalThis, 'process', {
-    __proto__: null,
-    value: process,
-    enumerable: false,
-    writable: true,
-    configurable: true
-  });
```

Comment:
There probably should be a comment explaining where is `process` added to the global object.

Reply:
Done